docusaurus/docs/quickstarts/network/hosted.md

```diff
@@ -57,12 +46,20 @@ variable. For example:
 export EXTERNAL_DNS="ec2-18-100-100-100.us-east-2.compute.amazonaws.com"
 ```
 
+You may override the default administrator password "admin". The secret value will be stored in the `*.env` file inside the express install home directory so that it can be easily referenced by the `zitiLogin` function.

```

Comment:
I'd rather we just change the way expressInstall works to simply generate a password and ask the user if they want to use the generated one or some other password. why not just do that -- and then this doc section isn't needed?

Reply:
> A random password for user "admin" will be generated and stored in the `*.env` file inside the express install home directory so that it can be easily referenced by the `zitiLogin` function. Optionally, you may override with a preferred password by setting `ZITI_PWD` now.